src/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+export const getArticles = () => {
+  const baseUrl = `https://news-4iag.onrender.com/api/articles`;
+  return axios.get(baseUrl).then((response) => {
+    return response.data.articles;
+  });
+};
+
+export const getTopics = () => {
+  const baseUrl = `https://news-4iag.onrender.com/api/topics`;
+  return axios.get(baseUrl).then((response) => {
+    return response.data.topics;
+  });
```

Comment:
You can refactor these functions to use a custom instance, check out the axios notes for how to do this.